app/scripts/lib/oauth-client.js

```diff
@@ -1,59 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-define([
-  'lib/xhr',
-  'lib/oauth-errors'
```

Comment:
This is one thing I just caught. How are the error codes going to be handled? Is there overlap in the error codes currently returned by the oauth and auth servers? If so, we are going to have to have a migration plan. We are going to have a migration plan for anything querying for oauth errors anyways, because all of the oauth errors will need to be migrated to auth-errors.js